### PR TITLE
[N/A]: Misc Updates

### PR DESCRIPTION
**Pull request by mobinni**
**What does this PR do**
- Introduces prettier for consistent linting across editors
- Adds basic unit test for utils
- Cleans up some references and expectations around swagger configuration

**Why**
Because

**Important to note**
Should test in isolation for optional parameters to work, this change may break definition, but as there are not unit tests to validate the edge case of summary and description it's harder to test

### CODE REVIEW

File: src/swagger/auth.ts [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
+    },
+  },
+  security: [{ basicAuth: [] }],
 };
+
+const authenticationConfiguration = {
+  [AuthenticationType.BASIC]: basicConfig,
+};
+
+export const getAuthentication = (type?: 'basic') => authenticationConfiguration[type];
```
Comment (mobinni) [inline src/swagger/auth.ts]: could remove the need for a method call here by using an enum

Comment (knor-el-snor) [reply src/swagger/auth.ts]: That is very much correct sir

File: src/swagger/baseDefinition.ts [hunk 2]
```diff
@@ -8,17 +8,23 @@ import {
...
+}: ISwaggerBaseDefinition): ISwaggerDefinition => ({
+  host,
+  basePath,
+  swagger: apiVersion,
   info: {
     title,
     description,
-    version: '1.0.0',
-    contact: {
-      email: 'developers@shopbonsai.ca',
```
Comment (knor-el-snor) [inline src/swagger/baseDefinition.ts]: 😅
